### DISCUSSION TITLE
Crazyflie keep crashing

### DISCUSSION THREAD

**Discussion by sesem738** [bitcraze/discussions#1409] [Q&A]
I have the crazyflie with the flowdeck. Whenever I run the autonomousSequence example, after the initial ascent, it flips over crashes. I can however run the automonous_sequence_high_level.py. I have tried this on 3 different crazyflies and they all produce the same result

**Comment by gemenerik** [comment]
I just tried the autonomousSequence without problems. Can you verify you are on the latest released firmware and taking off from [a textured floor](https://www.bitcraze.io/2023/11/go-with-the-flow-relative-positioning-with-the-flow-deck/)?
